### PR TITLE
NOTICK: Use the correct password in the plugin

### PR DESCRIPTION
**Pull request by yift-r3**

### CODE REVIEW

File: tools/plugins/mgm/src/main/kotlin/net/corda/cli/plugins/mgm/BaseOnboard.kt [hunk 1]
```diff
@@ -46,6 +46,27 @@ abstract class BaseOnboard : Runnable {
...
+                "corda-initial-admin-user",
+                "--namespace",
+                cordaClusterName,
+                "-o",
+                "go-template={{ .data.password | base64decode }}"
+            ).start()
+            if (getSecret.waitFor() != 0) {
+                throw OnboardException("Can not get admin password. ${getSecret.errorStream.reader().readText()}")
+            }
+            getSecret.inputStream.reader().readText()
```
Comment (vkolomeyko) [inline tools/plugins/mgm/src/main/kotlin/net/corda/cli/plugins/mgm/BaseOnboard.kt:64]: Are we sure the full content of input stream will be available once the process has exited?

It was not the case for me on Windows and I had to capture `stdErr` and `stdOut` as shown in: `net.corda.applications.workers.rpc.cli.CliTask#execute`.

Comment (yift-r3) [reply tools/plugins/mgm/src/main/kotlin/net/corda/cli/plugins/mgm/BaseOnboard.kt:64]: I'm not sure this plugin works on Windows at all...

**Comment by vkolomeyko** [comment]
I can see how this PR is making an improvement to the existing code, so approving it.
However, going forward it might be a good idea to change this plugin to use `net.corda.cli.plugins.common.HttpRpcCommand` which provides a uniform way of for interaction with HTTP RPC server from CLI.

**Review by vkolomeyko — APPROVED** [review]

**Comment by charlieR3** [comment]
> I can see how this PR is making an improvement to the existing code, so approving it. However, going forward it might be a good idea to change this plugin to use `net.corda.cli.plugins.common.HttpRpcCommand` which provides a uniform way of for interaction with HTTP RPC server from CLI.

Created https://r3-cev.atlassian.net/browse/CORE-8612 to capture that. It's in a tech debt epic for now but we will be creating an epic for user experience which I think we can move this to. Thanks @vkolomeyko.

**Pull request merged**